src/collections/movie_list.js

```diff
@@ -0,0 +1,13 @@
+import Backbone from 'backbone';
+import Movie from '../models/movie';
+
+const MovieList = Backbone.Collection.extend({
+  model: Movie,
+  url: 'http://localhost:3000/movies',
+  parse(response) {
```

Comment:
Since this isn't doing anything different than the default, overriding `parse` is unnecessary.